test/unitest_solution_gen.cpp

```diff
@@ -18,23 +18,44 @@ namespace mmh3 {
 
 int main( int argc, const char** argv )
 {
-	FILE* f   = fopen( "test/pg1260.txt", "rb" );
-	FILE* out1 = fopen( "test/solution_hash32_seed0.txt", "wb" );
-	FILE* out2 = fopen( "test/solution_hash32_seed1234ABCD.txt", "wb" );
-	char buffer[2048];
+    FILE* f    = fopen( "test/pg1260.txt", "rb" );
+    FILE* out1 = fopen( "test/solution_hash32_seed0.txt", "wb" );
+    FILE* out2 = fopen( "test/solution_hash32_seed1234ABCD.txt", "wb" );
+    FILE* out3 = fopen( "test/solution_hash128_x86_seed0.txt", "wb" );
```

Comment:
is there really a reason to have solution files both for the x86 and the x64 implementations... if these aren't the same, isn't murmurhash broken?
I.e. it should be enough to have only one solution_hash128_***.txt